src/components/ErrorShare.jsx

```diff
@@ -0,0 +1,22 @@
+import styles from '../styles/emptyAndError'
+
+import React from 'react'
+import { translate } from 'cozy-ui/react/I18n'
+
+export const ErrorShare = ({ t, errorType }) => {
+  return (
+    <div className={styles['pho-errorShare']}>
+      <h2>{ t(`Error.${errorType}_title`) }</h2>
+      <p>{ t(`Error.${errorType}_text`)}</p>
+    </div>
+  )
+}
+
+const TranslatedError = translate()(ErrorShare)
```

Comment:
We often export translated component and we never call them `TranslatedSomething`, maybe we can rename this one in accordance?